laravel/resources/views/inc/standard-alerts.blade.php

```diff
@@ -0,0 +1,23 @@
+@if (session('success'))
+    <div class="row">
```

Comment:
[CLARIFICATION]

Why do we have here an extra `row`, while the other three `div`s don't have it? Is this wrong here? Same question for the following `col-lg` class. 🤔

Reply:
Thats right. All need the row and col div